doc/sphinx/source/introduction.rst

```diff
@@ -3,6 +3,14 @@ Introduction
 
 This documentation describes the usage and possibilities of Lua to extend Widelands.
 
+.. note::
+
+   Files of type ``.lua`` are very basic text files. Do not use any fancy word
+   processor (Word, OpenOffice and their like) because they produce the wrong 
+   file format. Make sure to use a plain text editor (like Notepad under 
+   Windows, nedit under Linux and TextEdit under Mac OS X). If you have
+   programmed before, you likely already have found your favorite editor...
```

Comment:
```suggestion
   programmed before, you likely already have found your favorite editor.
```

Reply:
Why so strict?

Reply:
An ellipsis looks out of place here to me. You can also keep it if you prefer it, but then please use the `'…'` character.
(`…` should be used instead of `...` everywhere except in monospaced code markup.)

Reply:
I decided to remove the last sentence.